product/ai-gateway/multimodal-capabilities/vision.mdx

```diff
@@ -208,12 +208,14 @@ Portkey's prompt library supports creating templates with image inputs. If the s
+Here are some of the key vision models across major LLM providers. Portkey supports all the latest models from these providers as they're released. For a complete and up-to-date list of supported models, please check the [provider-specific docs](../../../integrations/llms).  Please raise a [request](/integrations/llms/suggest-a-new-integration) or a [PR](https://github.com/Portkey-AI/gateway/pulls) to add model or provider to the AI gateway.
 
 | Provider                                             | Models                                                                                       | Functions              |
 | ---------------------------------------------------- | -------------------------------------------------------------------------------------------- | ---------------------- |
-| [OpenAI](/integrations/llms/openai)             | `gpt-4-vision-preview`                                                                         | Create Chat Completion |
-| [Azure OpenAI](/integrations/llms/azure-openai) | `gpt-4-vision-preview `                                                                        | Create Chat Completion |
-| [Gemini](/integrations/llms/gemini)             | `gemini-1.0-pro-vision `                                                                       | Create Chat Completion |
-| [Anthropic](/integrations/llms/anthropic)       | `claude-3-sonnet claude-3-haiku claude-3-opus `                                                | Create Chat Completion |
-| [AWS Bedrock](/integrations/llms/aws-bedrock)   | `anthropic.claude-3-sonnet anthropic.claude-3-haiku anthropic.claude-3-5-sonnet-20240620-v1:0` | Create Chat Completion |
+| [OpenAI](/integrations/llms/openai)             | `gpt-4-vision-preview`, `gpt-4o`, `gpt-4o-mini `                                                   | Create Chat Completion |
+| [Azure OpenAI](/integrations/llms/azure-openai) | `gpt-4-vision-preview`, `gpt-4o`, `gpt-4o-mini `                                                   | Create Chat Completion |
+| [Gemini](/integrations/llms/gemini)             | `gemini-1.0-pro-vision `, `gemini-1.5-flash`, `gemini-1.5-flash-8b`, `gemini-1.5-pro`              | Create Chat Completion |
+| [Anthropic](/integrations/llms/anthropic)       | `claude-3-sonnet`, `claude-3-haiku`, `claude-3-opus`,  `claude-3.5-sonnet`, `claude-3.5-haiku`     | Create Chat Completion |
+| [AWS Bedrock](/integrations/llms/aws-bedrock)   | `anthropic.claude-3-5-sonnet anthropic.claude-3-5-haiku anthropic.claude-3-5-sonnet-20240620-v1:0` | Create Chat Completion |
+
+For a complete list of all supported providers and models (including non-vision models), check out our [providers documentation](../../../integrations/llms).
```

Comment:
We should just say that we support all the vision models instead of again linking them to the llms page where they'll not find any such table